### PR TITLE
t: Add test description strings for all 'click_element_ok' calls

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: t/full-stack.t [hunk 1]
```diff
@@ -243,8 +243,8 @@ subtest 'cancel a scheduled job' => sub {
     $cancel_button[0]->click();
 };
 
-$driver->click_element_ok('All Tests',     'link_text');
-$driver->click_element_ok('core@noassets', 'link_text');
+$driver->click_element_ok('All Tests',     'link_text', 'All tests clicked to go to test 4');
```
Comment (Amrysliu) [inline t/full-stack.t:246]: how about 'Clicked All Tests to go to test 4' ?

Comment (okurz) [reply t/full-stack.t:246]: Hm, sounds like the same to me

Comment (kalikiana) [reply t/full-stack.t:246]: There is a difference in capitalization in Jane's suggestion.

How about also using "Button" here to avoid the ambiguity?

Comment (okurz) [reply t/full-stack.t:246]: Ah, I see. Yes, changed to 'All Tests'. I would avoid "Button" because it's basically a text link, no button.

**Pull request merged**